lib/server.js

```diff
@@ -244,14 +246,28 @@ internals.Server = class {
 
     ext(events, method, options) {        // (event, method, options) -OR- (events)
```

Comment:
This doesn't handle `(event, options)`, which the docs imply is possible.

Reply:
I'll clarify the docs.